src/structures/node.jl

```diff
@@ -2,14 +2,72 @@
 abstract type Node end
 Base.show(io::IO, n::Node) = print(io, "n_$(n.id)")
 
+"""
+`StorageBehavior` as supertype for individual storage behaviours.
+
+Storage behaviour is used to identify how a storage node should behave within the individual
+`TimeStructure`s of a strategic period.
+"""
+abstract type StorageBehavior end
+
+"""
+`Accumulating` as supertype for an accumulating storage level.
+
+Accumulating storage behaviour implies that the change in the overall storage level in a
+strategic period can be both positive or negative.
+
+Examples for potential usage of `Accumulating` are CO₂ storages in which the CO₂ is
+permanently stored or multi year hydropower magazines.
+"""
+abstract type Accumulating <: StorageBehavior end
```

Comment:
I find the term `Accumulating` a bit confusing as it seems to imply that storage level is only allowed to increase. Not sure what would be a better name, though, maybe something along the lines of `StrategicStorage` or `MultiSeasonal`?

Reply:
I agree, that `Accumulating` points towards that direction. I am not fully certain about the best name for it, `StrategicStorage` could hold, but on the other hand, this would be not short for a supertype either. I would say we take a bit of time to think through it and come up with a solution eventually before we merge. There are also additional major changes coming in with the allowance of `charge` and `discharge` capabilities.